fab/fabfile.py

```diff
@@ -436,6 +436,22 @@ def apply_patch(patchfile=None):
     silent_services_restart(use_current_release=True)
 
 
+@task
+def reverse_patch(patchfile=None):
+    """
+    Used to reverse a git patch created via `git format-patch`. Usage:
+
+        fab <env> reverse_patch:patchfile=/path/to/patch
+
+    Note: Only use this when absolutely necessary.
```

Comment:
Haha, the things we do for NIC... That's good if it fails in a non-destructive way. I don't have all the flags to `git apply` in my head, but I was thinking that it could end up in a state where the patch was partially applied and needed human intervention—that probably involves a flag you're not using here.